.github/workflows/docs-unstable.yml

```diff
@@ -50,11 +50,11 @@ jobs:
                 aws-region: ${{ secrets.AWS_REGION }}
                 role-to-assume: ${{ secrets.AWS_OIDC_RUNNER_ROLE }}
 
-        -   name: Copy redirect file
-            run: aws s3 cp --quiet ./docs/_build/html/_static/redirect.html s3://docs.pyscript.net/index.html
+        # -   name: Copy redirect file
+        #     run: aws s3 cp --quiet ./docs/_build/html/_static/redirect.html s3://docs.pyscript.net/index.html
 
-        # -   name: Delete latest directory
-        #     run: aws s3 rm --recursive s3://docs.pyscript.net/latest/
+        # -   name: Delete unstable directory
+        #     run: aws s3 rm --recursive s3://docs.pyscript.net/unsatble/
```

Comment:
I know it's commented out but this is a typo, might cause problems if someone blindly tries to uncomment it

Reply:
If we remove the folder the s3 sync command will recreate it, the only issue is that we would push the whole build into s3, but I think a lot of the code was copied from other actions so I'll clean it up in a follow up 😄